test/feralfile_exhibition_v3_3.js

```diff
@@ -0,0 +1,230 @@
+const FeralfileExhibitionV3_3 = artifacts.require('FeralfileExhibitionV3_3');
+
+const IPFS_GATEWAY_PREFIX = 'https://ipfs.bitmark.com/ipfs/';
+
+const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
+
+contract('FeralfileExhibitionV3_3', async (accounts) => {
+  before(async function () {
+    this.exhibition = await FeralfileExhibitionV3_3.new(
+      'Feral File V3_4 Test 001',
+      'FFV3',
+      1000,
+      '0x8fd310de32848798eB64Bd88f9C5656Eea32415e',
+      'https://ipfs.bitmark.com/ipfs/QmaptARVxNSP36PQai5oiCPqbrATvpydcJ8SPx6T6Yp1CZ',
+      IPFS_GATEWAY_PREFIX,
+      true,
+      true,
+      accounts[0]
```

Comment:
use owner as minter might not test the minter correctly. We can not distinguish an operation is allowed by minter or by owner.

Reply:
Updated